### ISSUE TITLE
activation on all tabs

### ISSUE THREAD

**Issue by panch47** [niutech/chrome-devtools-sidebar#2]
Hi, thank you for this extension. I recently moved to chrome, I was a user of mozilla. Is it possible to show this inspection bar to all tabs and at startup of chrome? Thanks

**Comment by nonobio** [comment]
Hi,
I agree. Is it possible to pin it permanently for all tabs ? Also, when i open a link on the actual tab, sidebar is closed, can i avoid this ?
Thanks

**Comment by niutech** [comment]
No, because dev tools are linked to a particular tab. That said, you can run `chrome.exe` with a switch `--auto-open-devtools-for-tabs` to open dev tools on each tab, but this may or may not work as intended.

**Comment by nonobio** [comment]
Thanks for your reply :)
